documentation/modules/operators/ref-operator-cluster-feature-gates.adoc

```diff
@@ -78,3 +78,37 @@ Using StrimziPodSets instead of StatefulSets provides more control over the func
 To enable the `UseStrimziPodSets` feature gate, specify `+UseStrimziPodSets` in the `STRIMZI_FEATURE_GATES` environment variable in the Cluster Operator configuration.
 
 IMPORTANT: The `UseStrimziPodSets` feature gate must be disabled when downgrading to Strimzi 0.27 and earlier versions.
+
+[id='ref-operator-use-kraft-feature-gate-{context}']
+== UseKRaft feature gate
+
+The `UseKRaft` feature gate has a default state of _disabled_.
+
+The `UseKRaft` feature gate deploys the Kafka cluster in the KRaft mode (without ZooKeeper).
+This feature gate is currently intended only for development and testing.
+
+IMPORTANT: **The KRaft mode is not ready for production in Apache Kafka or in Strimzi.**
+
+When the `UseKRaft` feature gate is enabled, the Kafka cluster will be deployed without ZooKeeper.
+The `.spec.zookeeper` section in the Kafka CR will be ignored, but still needs to be present.
+The API provided by the `UseKRaft` feature gate for configuring the Kafka cluster nodes and their roles is not final and is expected to change before the KRaft mode is production ready.
+
+Currently, the KRaft mode in Strimzi has several major limitations:
+
+* Moving from Kafka clusters with ZooKeeper to KRaft clusters or the other way around is not supported.
+* Upgrades / downgrades of Apache Kafka versions or of the Strimzi operator are not supported.
+  Users might need to delete the cluster, upgrade the operator and deploy a new Kafka cluster.
+* Entity Operator (both User and Topic operator) are not supported.
+  The `spec.entityOperator` section should be removed from the Kafka custom resource.
+* `simple` authorization is not supported.
+* SCRAM-SHA-512 authentication is not supported.
+* JBOD storage is not supported (the `type: jbod` storage can be used, but the JBOD array can contain only one disk).
+* Liveness and readiness probes are disabled.
+* All Kafka nodes have both the `controller` and `broker` KRaft roles.
+
+For more information about the `UseKRaft` feature gate, see the link:https://github.com/strimzi/proposals/blob/main/036-kraft-mode.md[KRaft mode proposal^]
```

Comment:
Do we need a link to the proposal? We don't normally link to proposals.  I'm not sure there's a lot more to learned from it, as we have the limitations here.

Reply:
Ok, removed.